examples/camera-controller/commands/common/RemoteDataModelLogger.cpp

```diff
@@ -42,9 +42,9 @@ RemoteDataModelLoggerDelegate * gDelegate;
 
 CHIP_ERROR LogError(Json::Value & value, const chip::app::StatusIB & status)
 {
-    if (status.mClusterStatus.HasValue())
+    if (status.mClusterStatus.has_value())
     {
-        auto statusValue          = status.mClusterStatus.Value();
+        auto statusValue          = *status.mClusterStatus;
```

Comment:
I guess this is just a question of code style, but I personally would prefer using `.value()` for accessing `std::optional` instead of `operator *` since it goes with `has_value`, and an optional is not a pointer-like type.

Reply:
with exceptions, value is "safe" and throws an exception. operator* does not check has_value and is faster/uses less flash, so if we pre-checked I prefer it for that reason (very very minute flash savings I believe in my tests, even though we do compile without exceptions in theory)